simulation_runbooks/v3/runbook_fast_check.json

```diff
@@ -1,25 +1,31 @@
 {
-    "nodes": 4,
-    "maxNodeServiceWorkers": 100,
-    "maxNodeServiceQueueSize": 1000,
-    "avgNetworkLatency": "100ms",
-    "rpcDetail": {
+    "node": {
```

Comment:
Should we rename this directory from `simulation_runbooks` to `simulation_plans`?

Reply:
Good catch.